package.json

```diff
@@ -2493,8 +2493,15 @@
         "fsevents": "^2.3.2"
     },
     "overrides": {
-        "simple-get": "3.1.1",
-        "fs-path": "0.0.25",
+        "simple-get@<3.1.1": "3.1.1",
+        "validator@<13.7.0": "13.7.0",
+        "jpeg-js@<0.4.4": "0.4.4",
```

Comment:
using `@<` so that any package that depends on a later version will still get the later version instead of what we specific here.
I.e. we need to ensure we pin only to a specific version unless absolutely necessary.